README.md

```diff
@@ -192,4 +198,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
```

Comment:
- [ ] It would be better if we implement test for User post show page as show in the screenshot below

![Screenshot from 2022-08-04 17-05-45](https://user-images.githubusercontent.com/46388113/182909797-0ee71cac-126d-4104-87b4-37dd955c01f9.png)

Reply:
Hello @addod19 ,
I have include all the test state in the picture above. See this please [code](https://github.com/evans22j/Blog-App/blob/275824e473eae52f9a709881648fe222127c51e2/spec/integration/posts_show_spec.rb#L28-L60)